tempto-core/src/main/java/com/teradata/tempto/internal/DefaultRequirementsCollector.java

```diff
@@ -42,19 +46,47 @@
 
     public DefaultRequirementsCollector(Configuration configuration)
     {
-        this.collectors = ImmutableList.of(new RequirementsFromAnnotationCollector(configuration));
+        this.collectors = ImmutableList.of(
+                new RequirementsFromAnnotationCollector(configuration),
+                new CommandRequirementsCollector(configuration));
```

Comment:
Maybe better name for this one would be `CommandRequirementsFromConfigurationCollector`